.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -11,6 +11,8 @@ Feel free to ask questions in #prometheus-operator on Kubernetes Slack!
 Note: This repository is about prometheus-operator itself, if you have questions about:
 - helm installation, go to https://github.com/helm/charts repository
```

Comment:
We should change that link too. prometheus-operator helm chart is now hosted in https://github.com/prometheus-community/helm-charts